app/src/renderer/components/staking/ModalStake.vue

```diff
@@ -0,0 +1,112 @@
+<template lang="pug">
+  .modal-stake
+    .stake-header
+      img.icon(class='stake-atom' src="~assets/images/cosmos-logo.png")
+      span.tm-modal-title Stake
+
+    tm-form-group.stake-form-group(
+      :error='$v.amount.$invalid'
+      field-id='amount'
+      field-label='Amount'
+    )
+      tm-field#amount(
+        type="number"
+        v-model="amount")
+      tm-form-msg(name='Amount' type='between' :min='1' :max='maximum'
+        v-if='$v.amount.$invalid')
+
+    tm-form-group.stake-form-group(
+      field-id='to' field-label='To')
+      tm-field#to(v-model="to")
+
+    tm-form-group.stake-form-group(
+      field-id='from' field-label='From')
+      tm-field#from(
+        type="select"
+        :options="fromOptions"
+        v-model="fromIndex"
+      )
+
+    .stake-footer
+      tm-btn(
+        @click.native="onStake"
+        color="primary"
+        :disabled="$v.amount.$invalid"
+        size="lg"
+        value="Stake"
+      )
+</template>
+
+<script>
+import { between } from "vuelidate/lib/validators"
+import Modal from "common/TmModal"
+import { TmBtn, TmField, TmFormGroup, TmFormMsg } from "@tendermint/ui"
+
+export default {
+  props: [`fromOptions`, `maximum`, `to`],
+  components: {
+    Modal,
+    TmBtn,
+    TmField,
+    TmFormGroup,
+    TmFormMsg
+  },
+  data: () => ({
+    amount: 1,
+    fromIndex: 0
+  }),
+  validations() {
+    return {
+      amount: {
+        between: between(1, this.maximum)
+      }
+    }
+  },
+  methods: {
+    close() {
+      this.$emit("update:showModalStake", false)
+    },
+    onStake() {
+      this.$emit(`stake`, {
+        amount: this.amount,
+        from: this.fromOptions[this.fromIndex]
+      })
+
+      this.close()
+    }
+  }
+}
+</script>
+
+<style lang="stylus">
+@import '~variables'
+
+.modal-stake
+  background: var(--app-nav)
+  display: flex
+  flex-direction: column
+  height: 50%
+  justify-content: space-between
+  left: 50%
+  padding: 3%
```

Comment:
interesting choice of percentage on padding, i usually think of this being fixed around a scaling element. we've probably never established a design standard for styling but curious about your decision, penny for your thoughts?

Reply:
My mind is an empty canvas on this one and wide open to suggestions.

Reply:
I would go for `padding 1rem` or something like that.